### ISSUE TITLE
Exception in ionNavBackButton - TypeError: Cannot read property 'href' of null

### ISSUE THREAD

**Issue by nekojira** [meteoric/meteor-ionic#155]
I have an exception thrown when rendering a template that has a ionNavBackButton. 

It appears in Chrome console regardless if I load the route with that template directly or incoming from another view. 

Perhaps something is wrong with the client history since it references a href property value? The behavior of the back button is unaffected and I'm able to "go back" to the view I came from in my client. The exception doesn't break js execution either.

    Exception from Tracker afterFlush function:
    	debug.js:41 TypeError: Cannot read property 'href' of null
		    at Template.ionNavBackButton.rendered (ionNavBackButton.js:27)
		    at template.js:116
		    at Function.Template._withTemplateInstanceFunc (template.js:437)
		    at fireCallbacks (template.js:112)
		    at null.<anonymous> (template.js:205)
		    at view.js:104
		    at Object.Blaze._withCurrentView (view.js:523)
		    at view.js:103
		    at Object.Tracker._runFlush (tracker.js:468)
		    at onGlobalMessage (setimmediate.js:102)E

**Comment by tombaaaaa** [comment]
I keep getting the same error

**Comment by jamielob** [comment]
This looks to be fixed in the latest version on GitHub for me, but not yet in the version on Atmosphere. 
@nickw - Any chance you can push the latest version onto Atmosphere so I can test?

**Comment by lorensr** [comment]
Same here, I'm guessing due to https://github.com/meteoric/ionicons-sass/issues/5 ?

**Comment by nicholasalanbrown** [comment]
@nickw agreed, I get this error using the Atmosphere package. Can you push an update?

**Comment by baconmania** [comment]
Same symptoms here. I get the same stack trace, but navigation still works via my ionNavBackButton.

**Comment by baconmania** [comment]
My issue turned out to be at this line: https://github.com/meteoric/meteor-ionic/blob/d3de1c4f356aad10bf7edccf8960509f11c99611/components/ionNavBackButton/ionNavBackButton.js#L43

I was passing a `path` attribute on my `{{> ionNavBackButton }}` which was slightly wrong. I was passing `somePath/`, where I needed to pass `somePath`, so `backRoute` was undefined. If you do `Object.keys(Router.routes)`, you can see the exact names of the available routes.
